Examples/src/main/kotlin/com/mapbox/vision/examples/SafetyActivityKt.kt

```diff
@@ -45,16 +46,20 @@ class SafetyActivityKt : BaseActivity() {
             // current speed of our car
             val mySpeed = vehicleState.speed
 
-            // display toast with overspeed warning if our speed is greater than maximum allowed speed
             if (mySpeed > maxAllowedSpeed && maxAllowedSpeed > 0) {
                 // all VisionListener callbacks are executed on a background thread. Need switch to a main thread
                 runOnUiThread {
+                    // show current speed limits in view
+                    speed_alert_view.visibility = View.VISIBLE
+                    // notify user by Toast that he's overspeeding
                     Toast.makeText(
```

Comment:
Or - remove toast completely? I dont think we need two separate indications. Also - iOS does not show toasts as well

Reply:
Will remove toast and show 2 types of speed alert: normal and overspeeding